index.html

```diff
@@ -277,6 +277,9 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 															<span>TVM (Preview)</span></div>		
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="VitisAI">
 															<span>Vitis AI (Preview)</span></div>	
+														<div class="col-lg-2dot5 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="Xnnpack">
```

Comment:
The width of this item is different from all others. Is there a way to make them 5 to a row? Maybe col-lg-2dot4 on all items will fix this? The ordering of the EPs also need to be updated - all "Preview" ones should be listed AFTER the non-preview EPs, and each section (preview/non-preview) should be alphabetical. Please move SNPE higher up and add (Preview) next to XNNPACK

Reply:
`col-lg-2dot4` doesn't works neigher, if we gives 6 cols in a row, it will be looks scruffy

Reply:
The list of EPs should be alphabetical (current version needs to be updated as well). Can you please move NNAPI after DirectML, and move XNNPACK to the end along with "(Preview)?